Packs/MicrosoftExchangeOnline/ReleaseNotes/1_5_28.md

```diff
@@ -0,0 +1,5 @@
+#### Integrations
+
+##### EWS O365
+
+- Documentation and metadata improvements.
```

Comment:
```suggestion
Documentation and metadata improvements.
```
Should you mention in the RN that Impersonation has been deprecated by Microsoft?